Makefile

```diff
@@ -31,6 +31,10 @@ E2E_TEST_ARGS += -test.v -test.timeout=40m -ginkgo.v -ginkgo.slowSpecThreshold=6
 ifdef E2E_TEST_FOCUS
 	E2E_TEST_ARGS +=  -ginkgo.focus $(E2E_TEST_FOCUS)
 endif
+# skip ovs tests for non k8s providers
+ifeq (,$(findstring k8s-,$(KUBEVIRT_PROVIDER)))
+	E2E_TEST_SKIP += .*OVS.*
```

Comment:
You sure this works? On other places we extend the flag with `+=  FOO` (two spaces, one is splitting from `+=` another adding extra space to the flag.

Reply:
I tried it, by replacing k8s with blah and it worked as expected.